Solution1/WeatherApi/Constants.Connection.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace WeatherApi
+{
+    public static partial class Constants
+    {
+        public static class Connection
```

Comment:
remove

Reply:
retrieve connection string without this class, you need only one string not whole object

Reply:
+

Reply:
remove, you don't use it anywhere, only in startup, even in test you don't use it